### PR TITLE
Vcs/user guide] 14. 建立資料磁碟

### PR DESCRIPTION
**Pull request by ting88510**
#146

### CODE REVIEW

File: VCS/1. User Guides/14. 建立資料磁碟.md [hunk 1]
```diff
@@ -1 +1,95 @@
...
+### Commands
+
+```bash
+$ twccli mk vcs [-n]   #Data disk name
+                [-sz]  #Data disk size (GB)
+                
+```
+:::info
+{%hackmd @twccdocs/cli-parameter-note-en %}
+2. If you do not enter the parameter of the disk, it will be created with the default settings:
```
Comment (VigaWei) [inline VCS/1. User Guides/14. 建立資料磁碟.md:75]: [SUG]

enter the parameter of the disk > specify the optional parameters, the disk will be created with default settings

File: VCS/1. User Guides/14. 建立資料磁碟.md [hunk 2]
```diff
@@ -1 +1,95 @@
...
+
+| Disk type | Name |Disk size|
+| -------- | -------- | -------- | 
+| HDD  | twccli  | 100 GB  | 
+:::
+
+</div>
+
+### For example
```
Comment (VigaWei) [inline VCS/1. User Guides/14. 建立資料磁碟.md:84]: [SUG]

Examples

**Review by VigaWei — COMMENTED** [review]
Thx! Please revise the doc according to the comments.

**Comment by ting88510** [comment]
already revised, thanks~~~

**Pull request merged**